src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
```

Comment:
Consider writing a descriptive header comment for Deadline class? I noticed that you may have missed adding header comments for some other classes as well, so perhaps it would be good to consistently write descriptive header comments for all public classes/methods.